<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuperBoxClass</class>
 <widget class="QMainWindow" name="SuperBoxClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>478</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>480</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>232</red>
        <green>183</green>
        <blue>135</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>232</red>
        <green>183</green>
        <blue>135</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Pixel Number</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>SuperBox</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/myfirstscene1.bmp);</string>
   </property>
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>96</width>
      <height>64</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/image/mfc_Button2.bmp);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Bag_BackGround">
    <property name="geometry">
     <rect>
      <x>357</x>
      <y>52</y>
      <width>280</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Bag.bmp);</string>
    </property>
   </widget>
   <widget class="QFrame" name="slot4">
    <property name="geometry">
     <rect>
      <x>577</x>
      <y>70</y>
      <width>40</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Inventory_Slot2.bmp);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="slot3">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>70</y>
      <width>40</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Inventory_Slot2.bmp);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="slot8">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>115</y>
      <width>40</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Inventory_Slot2.bmp);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="slot2">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>70</y>
      <width>41</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Inventory_Slot2.bmp);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="slot9">
    <property name="geometry">
     <rect>
      <x>577</x>
      <y>115</y>
      <width>40</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Inventory_Slot2.bmp);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="slot6">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>115</y>
      <width>40</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Inventory_Slot2.bmp);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="slot1">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>70</y>
      <width>40</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Inventory_Slot2.bmp);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="slot5">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>115</y>
      <width>40</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Inventory_Slot2.bmp);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="slot7">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>115</y>
      <width>41</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Inventory_Slot2.bmp);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="slot0">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>70</y>
      <width>40</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Inventory_Slot2.bmp);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Fox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/24_bagICON.bmp);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="SelectUI">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>491</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/SelectUI.bmp);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Select_Button2">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>210</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/cake1.bmp);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Select_Button3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/cake1.bmp);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Select_Button1">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>210</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/cake1.bmp);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SlectButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>320</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <underline>false</underline>
      <stylestrategy>NoAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Bag.bmp);</string>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="SelectLabel">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>170</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/UILabel.bmp);</string>
    </property>
    <property name="text">
     <string>Choose One</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="PropertiesUI">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>48</width>
      <height>196</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/PropertiesUI.bmp);
</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Property_Attack">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>140</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Property_Attack.bmp);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Property_AttackLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>180</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Attack</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Property_Text.bmp);</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>Property_Attack</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="Property_DefenceLabel">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>240</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Defence</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Property_Text.bmp);</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>Property_Defence</cstring>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Property_Defence">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>200</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Property_Denfence.bmp);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Property_HPLabel">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>300</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string notr="true">Health</string>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Property_Text.bmp);</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>Property_HP</cstring>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Property_HP">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>260</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Property_HP.bmp);</string>
    </property>
   </widget>
   <widget class="QLabel" name="DesCriptonBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>240</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Description_Box.bmp);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>(●'◡'●)</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>9</number>
    </property>
    <property name="indent">
     <number>2</number>
    </property>
   </widget>
   <widget class="QGraphicsView" name="BattleWindow">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>850</width>
      <height>380</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/BattleWindow.bmp);
</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Player">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>96</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/mfc_Button2.bmp);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="PowerUI">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>76</width>
      <height>76</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/PowerUI.bmp);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="BulletSlot">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/BulletSlot.bmp);

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveIcon">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>210</red>
          <green>181</green>
          <blue>205</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>213</red>
          <green>200</green>
          <blue>212</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>169</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>210</red>
          <green>181</green>
          <blue>205</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>213</red>
          <green>200</green>
          <blue>212</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>169</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>213</red>
          <green>200</green>
          <blue>212</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Pixel Number</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/LoadSystem/Save_Icon.bmp);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <zorder>Fox</zorder>
   <zorder>SelectUI</zorder>
   <zorder>Select_Button2</zorder>
   <zorder>Select_Button3</zorder>
   <zorder>Select_Button1</zorder>
   <zorder>SlectButton</zorder>
   <zorder>SelectLabel</zorder>
   <zorder>PropertiesUI</zorder>
   <zorder>Property_Attack</zorder>
   <zorder>Property_AttackLabel</zorder>
   <zorder>Property_DefenceLabel</zorder>
   <zorder>Property_Defence</zorder>
   <zorder>Property_HPLabel</zorder>
   <zorder>Property_HP</zorder>
   <zorder>startButton</zorder>
   <zorder>DesCriptonBox</zorder>
   <zorder>BattleWindow</zorder>
   <zorder>Bag_BackGround</zorder>
   <zorder>slot0</zorder>
   <zorder>slot4</zorder>
   <zorder>slot6</zorder>
   <zorder>slot2</zorder>
   <zorder>slot7</zorder>
   <zorder>slot5</zorder>
   <zorder>slot8</zorder>
   <zorder>slot9</zorder>
   <zorder>slot1</zorder>
   <zorder>slot3</zorder>
   <zorder>Player</zorder>
   <zorder>PowerUI</zorder>
   <zorder>BulletSlot</zorder>
   <zorder>SaveIcon</zorder>
  </widget>
  <action name="actionGame">
   <property name="text">
    <string>Game</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="superbox.qrc"/>
 </resources>
 <connections/>
</ui>
